extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -190,8 +193,9 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
   /** @override */
   onCreativeRender(isVerifiedAmpCreative) {
     super.onCreativeRender(isVerifiedAmpCreative);
-    injectActiveViewAmpAnalyticsElement(
-        this, this.ampAnalyticsConfig, this.responseHeaders_);
+    if (this.ampAnalyticsConfig_) {
+      insertAnalyticsElement(this.element, this.ampAnalyticsConfig_, true);
```

Comment:
https://github.com/ampproject/amphtml/pull/8628#discussion_r110455658
To avoid waiting for `#analyticsForDocOrNull`. Also calling `loadExtension` to already loaded extension is cheap, so I would rather put true here.